tox.ini

```
@@ -28,7 +28,7 @@ extras =
     all
     docs
 commands =
-    sphinx-build -j auto --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
+    sphinx-build --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
```

```suggestion
    sphinx-build -j auto --color -W --keep-going -b html -d _build/.doctrees . _build/html {posargs}
```
pydata fixed it but its not out, we can fail on this for now

---

I don't think it makes much difference here? it's not a big doc build project? I guess it wont break RTD tho.

---

Leave it for now then.